test/13_arraycommon.jl

```diff
@@ -2165,6 +2165,17 @@ end
         @test_throws BoundsError rp[-1]
         @test_throws BoundsError rp[end + 1]
         @test_throws MethodError similar(rp)
+
+        irp = DataAPI.invrefpool(y)
+        for lev in (eltype(y) >: Missing ? [missing; levels(y)] : levels(y))
+            @test isequal(rp[irp[lev]], lev)
+        end
+
+        @test_throws KeyError irp[1]
+        @test_throws KeyError irp[z]
```

Comment:
Where is `z` defined?

Reply:
it is not - I do not know why it is not `"z"` as in my local file :confused: